### DISCUSSION TITLE
RNode MHz options

### DISCUSSION THREAD

**Discussion by KnoAll** [markqvist/Reticulum#25] [Help & Questions] [answered]
I'm interested in purchasing an RNode from your shop but am not sure which MHz option to choose. I'm in the US, does locality have a bearing on this, or are there other factors that determine the best choice for a particular case?

**Comment by 4c3e** [comment]
The country you are in does matter when it comes to LoRa frequency bands. In the US LoRa runs in 902–928 MHz range. Online I've seen this range be referred to as the US915 zone.

**Comment by markqvist** [comment, marked as answer]
Yes, it is going to depend on what frequency band you want to use. If you are in the US, and want to use ISM bands, you should go with a 915 MHz version. Some people use private spectrum allocations within the lower UHF bands though, where the 420-525 MHz option is useful.

I would love to sell you one of my version of the RNode, but currently, I do not have sellable stock, due to the ever-notorious chip and semiconductor shortage.

You are better off getting one of the [supported devices](https://github.com/markqvist/RNode_Firmware#supported-hardware) and just installing the RNode firmware on them :)

**Comment by arizonajon** [comment]
> I'm interested in purchasing an RNode from your shop but am not sure which MHz option to choose. I'm in the US, does locality have a bearing on this, or are there other factors that determine the best choice for a particular case?

I am a resident in the US. I'm also a ham radio operator. There is certainly no reason not to concentrate on the 902-928 MHz band, as others have pointed out, it's under FCC Part 15 and Part 18 rules and is kind of a hodgepodge of uses from baby intercoms to high-power radio-thermal heaters and such. It's also included in the Part 97 amateur radio spectrum so there's plenty of experimentation that goes on there. However, it is a noisy band.

If you are a licensed ham, you have access to the 420-450 MHz (70 cm) band. But, our antiquated rules (see 47 CFR 97.311, 309)B), 307)f)6)) seem to restrict spread-spectrum modes like this to a 100 kHz bandwidth and a 56 k baud rate. So, it's my opinion that I can be completely legal with spreads up to 100 kHz and 56 kbaud rates. However, there are bandplans for different uses there so can't just plop down anywhere. Yes, there are other hams who suggest that the output power and duty cycles are so low, and the SS nature in itself spreads that small amount of power even more, so operation in much of the band would be fine. I haven't gone that far yet, but when I get some "434 MHz" units I am going to start searching for a quiet spot in the spectrum. Just avoid the weak signal part of the band around 430 MHz.

Cheers - Jon

**Reply by markqvist** [reply]
Thanks a lot for providing these detailed clarifications. From someone who actually knows the details on the legislation "over there" :)

**Comment by stevewa2066** [comment]
I use the 70 cm and I have had pretty good success with it. I’m in a
densely populated area with tons of 900 MHz stuff in it. So having sone
“clearer” space is nice.

I also chose it because I have lots of 70 cm antennas and didn’t want to
buy more.
My limited experience so far is the higher speed the shorter the distance.
Which, if you look at the spec, that’s what happens.

I have seen on small bi directional amp on eBay that would give you 2
watts, but have not tried it. Having a better antenna might be the way to
go and not kill the RX sensitivity.

What is your goal?

Steve N0FPF

On Thu, Apr 28, 2022 at 6:11 AM arizonajon ***@***.***> wrote:

> I'm interested in purchasing an RNode from your shop but am not sure which
> MHz option to choose. I'm in the US, does locality have a bearing on this,
> or are there other factors that determine the best choice for a particular
> case?
>
> I am a resident in the US. I'm also a ham radio operator. There is
> certainly no reason not to concentrate on the 902-928 MHz band, as others
> have pointed out, it's under FCC Part 15 and Part 18 rules and is kind of a
> hodgepodge of uses from baby intercoms to high-power radio-thermal heaters
> and such. It's also included in the Part 97 amateur radio spectrum so
> there's plenty of experimentation that goes on there. However, it is a
> noisy band.
>
> If you are a licensed ham, you have access to the 420-450 MHz (70 cm)
> band. But, our antiquated rules (see 47 CFR 97.311, 309)B), 307)f)6)) seem
> to restrict spread-spectrum modes like this to a 100 kHz bandwidth and a 56
> k baud rate. So, it's my opinion that I can be completely legal with
> spreads up to 100 kHz and 56 kbaud rates. However, there are bandplans for
> different uses there so can't just plop down anywhere. Yes, there are other
> hams who suggest that the output power and duty cycles are so low, and the
> SS nature in itself spreads that small amount of power even more, so
> operation in much of the band would be fine. I haven't gone that far yet,
> but when I get some "434 MHz" units I am going to start searching for a
> quiet spot in the spectrum. Just avoid the weak signal part of the band
> around 430 MHz.
>
> Cheers - Jon
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/markqvist/Reticulum/discussions/25#discussioncomment-2654638>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRMG4T4PIXNMEGKQYYN2YTVHKE6PANCNFSM5SHAQBNA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Pardon the brevity, sent from a mobile device. So there.

**Reply by arizonajon** [reply]
Hi Steve - My goal is mobile APRS, at least for now. I was intrigued by Mark Q's testing several years ago. In the limited testing I've been able to perform with the original rnodes, i saw ranges easily more than 6 km and I wasn't even trying. And that was in the 902-928 band. My desire would be to grow a population of rnode "digipeaters" and igates, and even up the throughput to something a bit more. 

But, so far, I found a bad receiver on one of my original rnodes, while the LilyGO units I've had horrible performance programming. Right now, they seem to be stuck permanently on the original freq/bw that I programmed them in first. Also, sometimes, even with the right settings, the stuff just doesn't communicate. It's a bit frustrating.

What are you all doing with the stuff in the 440 band? 

Cheers - Jon N7UV

**Comment by stevewa2066** [comment]
I have been doing APRS as well at around 1.2k. There are several other LORA
APRS projects around to try too. Most are based on Marks code! Some are all
inclusive, meaning you don’t have to have a PI or PC attached to it. Some
are doing digipeating. I have also tried the Meshtastic system as well.

Which brings up an interesting question of mesh APRS vs Digi APRS.

I would like to have a 56k link with a friend at some point just for fun.
Which is why I’m waiting for Reticulum. Probably have to add a massive heat
sink to the LoRA chip!

Steve



On Thu, Apr 28, 2022 at 9:00 AM arizonajon ***@***.***> wrote:

> Hi Steve - My goal is mobile APRS, at least for now. I was intrigued by
> Mark Q's testing several years ago. In the limited testing I've been able
> to perform with the original rnodes, i saw ranges easily more than 6 km and
> I wasn't even trying. And that was in the 902-928 band. My desire would be
> to grow a population of rnode "digipeaters" and igates, and even up the
> throughput to something a bit more.
>
> But, so far, I found a bad receiver on one of my original rnodes, while
> the LilyGO units I've had horrible performance programming. Right now, they
> seem to be stuck permanently on the original freq/bw that I programmed them
> in first. Also, sometimes, even with the right settings, the stuff just
> doesn't communicate. It's a bit frustrating.
>
> What are you all doing with the stuff in the 440 band?
>
> Cheers - Jon N7UV
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/markqvist/Reticulum/discussions/25#discussioncomment-2656002>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRMG4VLGZLJAMZZA5ITIRLVHKY3JANCNFSM5SHAQBNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Pardon the brevity, sent from a mobile device. So there.

**Reply by arizonajon** [reply]
have you tried working with the LilyGO units? I have a pair of the V2.1 "paxcounter" units. I wiped them using rnodeconf, autoinstalled rnode, and was able to get my 902 MHz frequency programmed in. However, I've had no end of challenges changing their initial settings, even by wiping and re-installing rnode. I don't know why. and on the specA, they always put out exactly +2dBm, no matter what input value I provide, even when the --info response shows that they're "programmed" correctly. 
What kind of distance are you trying to achieve with your 56k baud link?

**Reply by arizonajon** [reply]
And now that I think about it, I'd be happy with a low-bandwidth connection between rarely used 2m APRS digis and the greater Internet. I've got one digi in western AZ, almost atop a 5500' mountain, that hears a fair amount of traffic (mostly I-10 and US60, and some occasional jeeps roaming the desert) that prolly has 50 bits per second average. Having a 400-500 baud link to another better positioned site would be fine. All solar-powered and whatnot, but they need backhaul.

**Comment by stevewa2066** [comment]
That would be a great low cost use of these devices.

N0FPF

On Thu, Apr 28, 2022 at 6:04 PM arizonajon ***@***.***> wrote:

> And now that I think about it, I'd be happy with a low-bandwidth
> connection between rarely used 2m APRS digis and the greater Internet. I've
> got one digi in western AZ, almost atop a 5500' mountain, that hears a fair
> amount of traffic (mostly I-10 and US60, and some occasional jeeps roaming
> the desert) that prolly has 50 bits per second average. Having a 400-500
> baud link to another better positioned site would be fine. All
> solar-powered and whatnot, but they need backhaul.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/markqvist/Reticulum/discussions/25#discussioncomment-2658498>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRMG4TPUHH6DJT2KAPRI5DVHMYTHANCNFSM5SHAQBNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Pardon the brevity, sent from a mobile device. So there.
